models/allegro/test_allegro_kappa.py

```
@@ -0,0 +1,348 @@
+# Symmetry parameters
+symprec = 1e-5  # symmetry precision for enforcing relaxation and conductivity calcs
+enforce_relax_symm = True  # Enforce symmetry with during relaxation if broken
+# Conductivity to be calculated if symmetry group changed during relaxation
+conductivity_broken_symm = False
+save_forces = True  # Save force sets to file
+temperatures: list[float] = [300]
+displacement_distance = 0.03
+ignore_imaginary_freqs = True
+
+# Task splitting:
+slurm_nodes = int(os.getenv("SLURM_NNODES", 1))
```

Ruff (0.12.2)

Invalid type for environment variable default; expected `str` or `None`

(PLW1508)